css/style.css

```diff
@@ -67,11 +66,155 @@ body {
 }
 
 header {
+   background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+   height:500px;
+   padding: 25px 90px 0px 90px;
+}
+
+header h1 {
+  font-weight: 800;
+  font-size: 30px;
+  color:white;
+}
+
+header h2 {
+  font-size: 30px;
+  font-weight:700;
+  margin-top:65px;
+  color:white;
+  text-align: center;
+}
+
+.getbutton {
+  border-radius: 5px;
+  font-weight: 700;
+  background-color: yellow;
+  width: 220px;
+  height:50px;
+  font-size: 20px;
+  margin-top:40px;
+  margin-left:310px;
```

Comment:
@cliffjadoo once again add `display: block` to change it it from an  inline element. Then change your margins to this `margin-top: 40px; margin-left: auto; margin-right: auto;` and that will center it. You could also rewrite those rules like this `margin: 40px auto 0px` to make it less verbose.